variables.tf

```diff
@@ -4,6 +4,11 @@ variable "cluster_enabled_log_types" {
   type        = "list"
 }
 
+variable "cluster_log_retention_in_days" {
+  default     = "0"
```

Comment:
I think maybe a default of `30` or `90` might be more sensible. What do you think?

Reply:
Cloudwatch logs can get expensive and on a busy cluster there is a lot of messages. So maybe let's go with `90`.

Reply:
@max-rocket-internet makes sense

Reply:
@max-rocket-internet set default retention to 90 days